src/core/logger.js

```diff
@@ -16,9 +19,36 @@ if (process.env.NODE_ENV === 'test') {
   pino = require('pino');
 }
 
-const appName = config.get('appName');
-
-export default pino({
+const pinoLogger = pino({
   level: config.get('loggingLevel'),
-  name: appName,
+  name: config.get('appName'),
 });
+
+if (config.get('enableRequestID')) {
+  // eslint-disable-next-line global-require
+  httpContext = require('express-http-context');
```

Comment:
Now that you have the browser config in `package.json`, can this become a normal import?

Reply:
no, unfortunately because for some reasons (and assuming I remember correctly), the test suite is failing when we import the lib.